### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by didukhroma**
add section for 3 breakpoints

### CODE REVIEW

File: src/css/reviews.css [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+}
+/* SECTION LIST */
+@media only screen and (min-width: 768px) {
+  .reviews-list {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 32px;
+  }
+}
+/* @media only screen and (min-width: 1280px) {
```
Comment (asaulyak) [inline src/css/reviews.css:81]: remove commented out code

File: src/partials/reviews.html [hunk 2]
```diff
@@ -1 +1,104 @@
-<section>reviews</section>
+<!-- REVIEWS -->
+<section class="reviews">
+  <div class="container">
+    <h2 class="reviews-title">
+      What our <span class="reviews-title-accent">students</span> say
```
Comment (asaulyak) [inline src/partials/reviews.html:5]: use global `.accent` class

File: src/partials/reviews.html [hunk 3]
```diff
@@ -1 +1,104 @@
...
+                ./img/reviews/review-1/review-pic-1@2x.png  2x
+              "
+              src="./qimg/reviews/review-1/review-pic-1.png"
+              alt="student Anna"
+              loading="lazy"
+            />
+          </div>
+
+          <div class="reviews-item-thumb">
+            <h3 class="reviews-item-title">Anna</h3>
```
Comment (asaulyak) [inline src/partials/reviews.html:32]: use paragraph

File: src/partials/reviews.html [hunk 4]
```diff
@@ -1 +1,104 @@
...
+            <h4 class="reviews-item-subtitle">Student</h4>
```
Comment (asaulyak) [inline src/partials/reviews.html:33]: use paragraph

**Review by asaulyak — COMMENTED** [review]
tablets should display only 2 latest reviews
mobiles - only 1

**Comment by didukhroma** [comment]
fix all comments

**Review by asaulyak — APPROVED** [review]

**Pull request merged**